modules/floating-buttons/base/widget-contact-button-base.php

```diff
@@ -29,10 +29,13 @@ public function get_group_name(): string {
 	}
 
 	public function get_style_depends(): array {
-		if ( Plugin::$instance->experiments->is_feature_active( 'e_font_icon_svg' ) ) {
-			return parent::get_style_depends();
-		}
-		return [ 'elementor-icons-fa-solid', 'elementor-icons-fa-brands', 'elementor-icons-fa-regular' ];
+		$style_depends = Plugin::$instance->experiments->is_feature_active( 'e_font_icon_svg' )
+			? parent::get_style_depends()
+			: [ 'elementor-icons-fa-solid', 'elementor-icons-fa-brands', 'elementor-icons-fa-regular' ];
```

Comment:
@rami-elementor, WHERE THE ASSETS ARE REGISTERED?

Reply:
@KingYes In the icon manager.